src/context/authStore.test.tsx

```diff
@@ -52,21 +85,23 @@ describe("AuthStoreContextProvider", () => {
       </AuthStoreContextProvider>
     );
 
-    expect(mockGetItem).toHaveBeenCalledTimes(1);
-    expect(mockGetItem).toHaveBeenCalledWith("AUTH_STORE");
+    expect(mockReadBucket).toHaveBeenCalledTimes(1);
+    expect(mockReadBucket).toHaveBeenCalledWith("AUTH_STORE");
     expect(queryByTestId("loaded")).toBeNull();
 
     await waitFor(() => {
+      expect(queryByTestId("loaded")).toHaveTextContent("true");
```

Comment:
Noticed a minor change in order, did you encounter an issue with this expect being after?

Reply:
not a functional issue but more by convention and also readability of errors. The documentation explains `waitFor` as a function to get an element, so it actually returns whatever the inner function returns and the react native example does it that way with `getByTestId`. The only cosmetic issue with expect being after (inside the waitFor) is that if the `expect` fails, `waitFor` will keep running it until it timesout resulting in an error like `88 assertions instead of 4`